sklearn/cluster/tests/test_bicluster.py

```diff
@@ -21,7 +21,7 @@
 from sklearn.cluster.bicluster import _bistochastic_normalize
 from sklearn.cluster.bicluster import _log_normalize
 
-from sklearn.metrics import consensus_score
+from sklearn.metrics import (consensus_score, v_metric_score)
 
 from sklearn.datasets import make_biclusters, make_checkerboard
```

Comment:
The right name is `v_measure_score`

Reply:
thank you @jeremiedbb . Sorry for sloppiness, I was in a rush.